rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/client/VillainClient.java

```diff
@@ -60,4 +60,14 @@ public VillainClient(FightConfig fightConfig) {
 		return Uni.createFrom().completionStage(this::getRandomVillain)
 			.onFailure().retry().withBackOff(Duration.ofMillis(200)).atMost(3);
 	}
+
+  /**
+   * Calls hello on the Villains service.
+   * @return A "hello" from Villains
+   */
+  public String helloVillains() {
+    return this.villainClient.path("hello")
+      .request(MediaType.TEXT_PLAIN_TYPE)
+      .get(String.class);
```

Comment:
to make this return `Uni<String>` you can add `.rx(UniInvoker.class)` in the chain, like in the `getRandomVillain()` method.